scripts/src/components/organisation-form.js

```diff
@@ -98,12 +113,10 @@ var CreateOrg = React.createClass({
     if ('star_rating' in data) {
       data['star_rating'] = parseInt(data['star_rating'] || 0);
     }
-    data['reference_links'] = this.state.referenceLinks;
-    if (id) {
-      actions.updateOrganisation.push(_.extend(data, { organisationId: id }));
-    } else {
-      actions.createOrganisation.push(data);
-    }
+
+    data['reference_links'] = this._prepareReferenceLinks();
+    if (id) { actions.updateOrganisation.push(_.extend(data, { organisationId: id })); }
```

Comment:
I usually write it in one line if there is only 1 statement executed in the if statement but i don't feel strongly about it. I'll change it back if it makes it harder to read.